addons/binding/org.openhab.binding.mqtt.generic/README.md

```diff
@@ -242,7 +242,9 @@ Have a look at the following textual examples.
 
 ### A broker Thing with a Generic MQTT Thing and a few channels 
 
-demo.Things:
+Note: the [broker connection](https://www.openhab.org/addons/bindings/mqtt/) may be set up as a Bridge or as a stand-alone Thing. 
```

Comment:
I like to keep parts of this PR. But could you remove this sentence. It might just confuse.